Gemfile

```diff
@@ -43,6 +43,7 @@ end
 
 group :development do
   gem 'web-console' # Use console on exceptions pages [https://github.com/rails/web-console]
+  gem 'rubocop', require: false
```

Comment:
The `require: false` stops `Rubocop` from auto-loading since it's also a command-line tool, not a runtime dependency (like `rails` or `pg`). This keeps development efficient by not loading it with every `rails s`